### ISSUE TITLE
Can't create custom Hugo pattern

### ISSUE THREAD

**Issue by citadel-lewis** [freesewing/freesewing.org#695]
When I go to create custom Hugo hoodie with my measurements, it says "Oh no! There was an unexpected problem. Please report this."

Here's the error:

`Stack trace
Error: Error in Path.shiftAlong(): Ran out of path to shift along. Distance requested was 2, path length is0.37.
    at x.AxEy.x.shiftAlong (https://freesewing.org/component---src-pages-create-design-for-person-js-8d27888b7fdaab612c95.js:1:633915)
    at https://freesewing.org/component---src-pages-create-design-for-person-js-8d27888b7fdaab612c95.js:1:631316
    at x.AxEy.x.offset (https://freesewing.org/component---src-pages-create-design-for-person-js-8d27888b7fdaab612c95.js:1:631663)
    at o.d.draftFront (https://freesewing.org/component---src-pages-create-design-for-person-js-8d27888b7fdaab612c95.js:1:1051047)
    at o.It.draftFrontBase (https://freesewing.org/component---src-pages-create-design-for-person-js-8d27888b7fdaab612c95.js:1:192518)
    at o.AxEy.J.draft (https://freesewing.org/component---src-pages-create-design-for-person-js-8d27888b7fdaab612c95.js:1:656202)
    at T (https://freesewing.org/component---src-pages-patterns-pattern-index-js-60af9f5d7071a2661fdf.js:1:813344)
    at Ki (https://freesewing.org/framework-5a7bf077542a910ca662.js:1:70048)
    at vo (https://freesewing.org/framework-5a7bf077542a910ca662.js:1:116240)
    at cu (https://freesewing.org/framework-5a7bf077542a910ca662.js:1:108782)`

**Comment by joostdecock** [comment]
Can you share your username on the site please, so I can have a look?

thanks

**Comment by citadel-lewis** [comment]
Username is: citadel.lewis 
Thanks!

**Comment by joostdecock** [comment]
Hello again,

I had a look at this, and I suspect the issue is with your shoulder to shoulder measurement.

These measurements seem hard to reconcile:

- chest circumference: 36"
- shoulder to shoulder: 20"

The shoulder to shoulder measurement is close to half of the chest circumference. Since the chest circumference is the one that's easier to measure, I suggest you double check your shoulder to shoulder measurement. Documentation is here: https://freesewing.org/docs/measurements/shouldertoshoulder/

As a test, I updated your shoulder to shoulder measurement to 17" (down from 20") and sure enough, Hugo drafts fine that way.

I've set it back to 20" but I think you'll find that's where the problem is.
